includes/rrdtool.inc.php

```diff
@@ -88,9 +88,13 @@ function rrdtool_terminate() {
  * @param  array $rrd_pipes
  * @return integer
  */
-function rrdtool_pipe_close($rrd_process, &$rrd_pipes)
+function rrdtool_pipe_close(&$rrd_process, &$rrd_pipes)
```

Comment:
The & was what was removed last year to resolve some zombie processes - any reason to re-introduce it?